cellmlmanip/units.py

```diff
@@ -1,4 +1,7 @@
-"""Unit handling for CellML models, using the Pint unit library."""
+"""
+The :mod:`cellmlmanip.units` module provides unit handling for CellML models, using the
+`Pint unit library <https://pint.readthedocs.io/>`_.
```

Comment:
Should this be some other type of link, using the ones defined in conf.py?

Reply:
It could be, but I think this kind is fine. Not sure what a special Sphinx link would be for the whole library?